inst/rmarkdown/templates/iop/resources/template.tex

```diff
@@ -0,0 +1,122 @@
+%% Author template for Institute of Physics (IOP) Journal articles
+%% Adapted for rticles by Rob J Hyndman Rob.Hyndman@monash.edu. Dec 2021
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\documentclass[$if(fontsize)$$fontsize$$else$$if(twocolumn)$10pt$else$12pt$endif$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{iopart}
+
+%% BEGIN MY ADDITIONS %%
+\pdfminorversion=4
+\usepackage{hyperref, graphicx, booktabs}
+% Bibliography allowing natbib with Harvard
+\usepackage{har2nat}  % loads 'natbib' automatically
+\bibliographystyle{dcu}
+\providecommand\newblock{} % suppress some meaningless warnings
+
+$if(highlighting-macros)$
+% Pandoc syntax highlighting
+$highlighting-macros$
+$endif$
+
+% tightlist command for lists without linebreak
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
+$if(tables)$
+% From pandoc table feature
+\usepackage{array}
+$if(multirow)$
+\usepackage{multirow}
+$endif$
+\usepackage{calc} % for calculating minipage widths
+% Correct order of tables after \paragraph or \subparagraph
+\usepackage{etoolbox}
+$endif$
+
+$if(csl-refs)$
+% Pandoc citation processing
+\newlength{\cslhangindent}
+\setlength{\cslhangindent}{1.5em}
+\newlength{\csllabelwidth}
+\setlength{\csllabelwidth}{3em}
+\newlength{\cslentryspacingunit} % times entry-spacing
+\setlength{\cslentryspacingunit}{\parskip}
+% for Pandoc 2.8 to 2.10.1
+\newenvironment{cslreferences}%
+  {$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
+  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
+  {\par}
+% For Pandoc 2.11+
+\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
+ {% don't indent paragraphs
+  \setlength{\parindent}{0pt}
+  % turn on hanging indent if param 1 is 1
+  \ifodd #1
+  \let\oldpar\par
+  \def\par{\hangindent=\cslhangindent\oldpar}
+  \fi
+  % set entry spacing
+  \setlength{\parskip}{#2\cslentryspacingunit}
+ }%
+ {}
+\usepackage{calc}
+\newcommand{\CSLBlock}[1]{#1\hfill\break}
+\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
+\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
+\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
+$endif$
+
+%% END MY ADDITIONS %%
+
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+
+%%%%%%%%%%%%%%%%
+\begin{document}
+%%%%%%%%%%%%%%%%
+
+\title$if(shorttitle)$[$shorttitle$]$endif${$title$}
+
+$for(authors)$
+\author{$authors.name$$if(authors.footnote)$\footnote{$authors.footnote$}$endif$}
+$if(authors.address)$\address{$authors.address$}$endif$
+$if(authors.email)$\ead{$authors.email$}$endif$
+$endfor$
+
+\begin{abstract}
+$abstract$
+\end{abstract}
+
+$if(msc)$\ams{$msc$}$endif$
+
+\vspace{0.5cm}\noindent\textit{Keywords}: $for(keywords)$$keywords$$sep$, $endfor$
+
+\maketitle
+
+$if(twocolumn)$\ioptwocol$endif$
+
+$for(include-before)$
+$include-before$
+
+$endfor$
```

Comment:
To be useful, this should be placed before  in the template. When just above `$body$` there is no difference between including a `include-before` or writing in the Rmd document first. 

From Pandoc's doc: 
> Include contents of FILE, verbatim, at the beginning of the document body (e.g. after the <body> tag in HTML, **or the \begin{document} command in LaTeX**).

So I think this should be place either before the `\title` or between Authors and abstract. Even if this is not to be used. Otherwise, we could not include it.